CHANGELOG.md

```diff
@@ -78,6 +78,7 @@ Breaking changes:
   If it does not have a reference to `gen-webpack.config.js` then it will be regenerated.
 - [debug] removed `@theia/json` dependency. Applications should explicitly depend on `@theia/json` instead [#6647](https://github.com/eclipse-theia/theia/pull/6647)
 - [preferences] removed `@theia/json` dependency. Applications should explicitly depend on `@theia/json` instead [#6647](https://github.com/eclipse-theia/theia/pull/6647)
+- [plugin-ext] only keep at most 10 last session logs sub-folders.
```

Comment:
[minor] do you mind updating this entry?
1. the entry should not be placed under `Breaking Changes`
2. the entry should reference the corresponding pull-request like other entries
3. it should be updated to something more descriptive, for example:
```md
- [plugin-ext] added functionality to remove session logs from sub-folders older than 10 sessions [#6956](https://github.com/eclipse-theia/theia/pull/6956)
```

Reply:
Sure